docs/console/auth/import.md

```diff
@@ -0,0 +1,57 @@
+---
+title: Use an existing Amazon Cognito user pool and identity pool
```

Comment:
Very long title.. it'll show up weirdly in the sidenav. Suggestion 'Import Amazon Cognito resources'